db/schema.rb

```diff
@@ -17,8 +17,10 @@
 
   create_table "tweets", force: :cascade do |t|
     t.text "message", null: false
```

Comment:
Hm.. Think about how many characters Twitter allows its users in their tweets, and perhaps reconsider the use of this datatype for message 🙂